### PR TITLE
Single tap to zoom out. Single tap or drag to dismiss.

### PR DESCRIPTION
**Pull request by brianeunilkim**
Fix. 
Hides image view before zoom-in transition and shows it after zoom-out
transition. This prevents still image showing behind the zoom-in transition.

Update.
Instead of single tap to dismiss while zoomed in, single tap will zoom
back out. When zoomed out, single tap or draggin will dismiss. Facebook style.

### CODE REVIEW

File: Vertigo/TGRImageViewController.m [hunk 1]
```diff
@@ -44,6 +44,9 @@ - (void)viewDidLoad {
     
     [self.singleTapGestureRecognizer requireGestureRecognizerToFail:self.doubleTapGestureRecognizer];
     self.imageView.image = self.image;
+    self.scrollView.delegate = self;
```
Comment (gonzalezreal) [inline Vertigo/TGRImageViewController.m]: The view controller is already setup as the scroll view delegate in the XIB. This line can be removed.

File: Vertigo/TGRImageViewController.m [hunk 2]
```diff
@@ -44,6 +44,9 @@ - (void)viewDidLoad {
...
+    self.scrollView.bounces = YES;
+    self.scrollView.alwaysBounceVertical = YES;
```
Comment (gonzalezreal) [inline Vertigo/TGRImageViewController.m]: Please make these changes in the XIB instead on code.

**Comment by gonzalezreal** [comment]
Thanks for the fix on hiding the images. As for the behaviour changes, nothing against it, apart from a couple of comments I did on the code.

**Comment by brianeunilkim** [comment]
Scroll view vertical bounce setting moved to xib file. Removed unnecessary line.
Thanks!

**Comment by gonzalezreal** [comment]
LGTM

**Comment by gonzalezreal** [comment]
Thanks for contributing.

**Pull request merged**